src/controllers/IllnessStartDateController.ts

```diff
@@ -0,0 +1,64 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+
+import { BaseController } from 'app/controllers/BaseController';
+import { StartDateValidator } from 'app/controllers/validators/StartDateValidator';
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { Reasons } from 'app/models/Reasons';
+import { Feature } from 'app/utils/Feature';
+import { ILLNESS_START_DATE_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template: string = 'illness/illness-start-date';
+
+const navigation: Navigation = {
+    previous(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    },
+    next(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    }
+};
+
+const applyPadding = (dayMonth: string): string => {
+    return dayMonth.length < 2 ? dayMonth.padStart(2, '0') : dayMonth;
```

Comment:
That condition is not needed as per screenshot below:

<img width="183" alt="image" src="https://user-images.githubusercontent.com/1724691/88917746-3076b200-d260-11ea-82b2-4821832a413b.png">

Reply:
removed the padding - just thought it would look nicer when retrieving date from the session